src/Components/CardSmallChart.js

```diff
@@ -0,0 +1,99 @@
+import React from 'react';
+import { Line } from 'react-chartjs-2';
+import { useParams } from 'react-router-dom';
+import {
+  Chart as ChartJS,
+  CategoryScale,
+  LinearScale,
+  PointElement,
+  LineElement,
+  Filler,
+} from 'chart.js';
+import { useSelector } from 'react-redux';
+
+ChartJS.register(CategoryScale, LinearScale, PointElement, LineElement, Filler);
+const CardSmallChart = () => {
+  let coin;
+  let bcolor;
+  let color;
+  const { id } = useParams();
+  const crypto = useSelector((state) => state.cryptos);
+  crypto.forEach((element) => {
+    if (element.uuid === id) {
+      coin = element;
+    }
+  });
```

Comment:
- [ ] Amazing implementation of the chart! great job!

Reply:
Thank you so much 🖤 i appreciate it